src/variants.ini

```diff
@@ -853,3 +853,428 @@ stalemateValue = loss
+startFen = sgkgs/5/1ppp1/1PPP1/5/SGKGS[LNln] w 0 1
+lance = l
+shogiKnight = n
+promotedPieceType = l:g n:g
+
+# Dragonfly
+# Simplified Crazyhouse. Two rules have been removed from the original definition (Removing captured pawns and special pawn promotion). Credits to Procyon for the definition.
+# https://en.wikipedia.org/wiki/Dragonfly_(chess_variant)
+[dragonfly:chess]
+maxFile = 7
+maxRank = 7
+pocketSize = 7
+castlingKingsideFile = f
+castlingQueensideFile = b
+pieceDrops = true
+capturesToHand = true
+promotionRank = 7
+promotionPieceTypes = nbr
+doubleStep = false
+dropNoDoubled = p
+dropNoDoubledCount = 0
+startFen = rbbknnr/ppppppp/7/7/7/PPPPPPP/RBBKNNR[] w - - 0 1
+
+# Kamikaze Rooks
+# The objective of the game is to lose both rooks. Credits to Procyon for the definition.
+# Variant defined in Pychess discord and previously playable with Chessboi bot.
+[kamikazerooks:chess]
+extinctionValue = win
+extinctionPieceTypes = r
+
+# Mounted
+# Variant invented by Procyon. Credits to him for the definition.
+# Variant defined in Pychess discord and previously playable with Chessboi bot.
+[mounted]
+maxRank = 7
+maxFile = 7
+pieceDrops = true
+capturesToHand = true
+stalemateValue = loss
+nFoldValue = loss
+extinctionValue = loss
+extinctionPseudoRoyal = true
+extinctionPieceTypes = k
+flagPiece = k
+flagRegionWhite = *7
+flagRegionBlack = *1
+centaur = k
+bishop = b
+rook = r
+cannon = c
+customPiece1 = a:mBcpB 
+customPiece2 = s:FsfW
+startFen = 1cbkra1/2s1s2/7/7/7/2S1S2/1CBKRA1 w - - 0 1
+
+# Racing Chess
+# Win by campmate. No checks allowed.
+# Variant defined in Pychess discord and previously playable with Chessboi bot.
+[racingchess:chess]
+flagPiece = k
+flagRegionWhite = *8
+flagRegionBlack = *1
+flagMove = true
+checking = false
+
+# Shinobi Mirror
+# Symmetric version of Shinobi.
+# Variant defined in Pychess discord and previously playable with Chessboi bot.
+[shinobimirror:crazyhouse]
+commoner = c
+bers = d
+archbishop = j
+fers = m
+shogiKnight = h
+lance = l
+promotionRank = 7
+promotionPieceTypes = -
+promotedPieceType = p:c m:b h:n l:r
+mandatoryPiecePromotion = true
+stalemateValue = loss
+perpetualCheckIllegal = true
+startFen = lh1ck1hl/pppppppp/8/8/8/8/PPPPPPPP/LH1CK1HL[LHMMDJlhmmdj] w - - 0 1
+capturesToHand = false
+whiteDropRegion = *1 *2 *3 *4
+blackDropRegion = *5 *6 *7 *8
+immobilityIllegal = true
+flagPiece = k
+flagRegionWhite = *8
+flagRegionBlack = *1
+castling = false
+
+# Two Kings v2
+# Slightly different implementation of Two Kings chess.
+# Variant defined in Pychess discord and previously playable with Chessboi bot.
+[twokings2:chess]
+king = -
+commoner = k
+extinctionValue = loss
+extinctionPseudoRoyal = true
+extinctionPieceTypes = k
+extinctionPieceCount = 1
+startFen = rnbkkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBKKBNR w - - 0 1
+
+# Anti-Antichess
+# Lose at anti-chess to win at anti-antichess.
+# Variant defined in Liantichess website. Credits to SriMethan for the definition.
+[anti_antichess:giveaway]
```

Comment:
I was so far always using dashes as separators in variant names, so for consistency would suggest the same here.